src/pages/api/opt-out-feedback.ts

```diff
@@ -1,10 +1,14 @@
 import Cors from 'cors'
 import { NextApiRequest, NextApiResponse } from 'next'
-import {
-  GoogleSpreadsheet,
-  GoogleSpreadsheetRow,
-  GoogleSpreadsheetWorksheet,
-} from 'google-spreadsheet'
+import { GoogleSpreadsheet } from 'google-spreadsheet'
+
+// Filter the body for any keys that aren't included in this list
+export const allowedKeys = [
```

Comment:
Moved this def up so I could use it in my type definition for the req handler.

Reply:
👍 Nice improvement!